politeiad/config.go

```diff
@@ -441,6 +492,19 @@ func loadConfig() (*config, []string, error) {
 	// duplicate addresses.
 	cfg.Listeners = normalizeAddresses(cfg.Listeners, port)
 
+	// Load configurable mime types to politeiad.
+	if len(cfg.MimeTypes) == 0 {
+		cfg.MimeTypes = defaultMimeTypes
```

Comment:
The default should be set with the rest of the defaults at the beginning of `loadConfig`.